test/mock/items.js

```diff
@@ -17,6 +17,18 @@ module.exports = [
     title: "title",
     tool: "tool1",
     foo: "bar",
+    dynamicSchema: {
```

Comment:
This seems a little misleading to me as we do not store the schemas on the item in the database. By reading the tests one could have the impression that this is what we do though. Could you change this to store these schemas hardcoded in the tool mocks?

Reply:
Ah got it now.